.ci/Jenkinsfile

```diff
@@ -273,6 +273,10 @@ pipeline {
       }
     }
     stage('Coverage converge') {
+      when {
+        beforeAgent true
+        expression { return env.ONLY_DOCS == "false" }
```

Comment:
I was initially tempted to use `expression { return rubyDownstreamJobs.size() > 0 }` but I guess we don't want to as if it's empty then something really bad has happened with the pipeline